andela/Ah-frontend-guardians
# #162685628 Users can Edit and Delete their articles

#### What does this PR do?
This PR enables users to edit and delete articles that they authored.

#### Description of Task to be completed?
* Implement update and delete functionality from the profile page
* design the User interface for performing the functionality

#### How should this be manually tested?
Use the appropriate screenshots provided to view the expected results
* Visit the url below:
https://ah-frontend-guardians-pr-18.herokuapp.com/

* signup and login
* click on the avatar profile avatar to view your profile page.
* If you don't have any articles you have to post some using postman.
```POST https://ah-backend-guardians-staging.herokuapp.com/api/articles/```

```
Note: Remember to use the surrent token after a login, get from the browser local storage
{
	"title": "",
	"description": "",
	"body": ""
}
```
* After posting, you can refresh the profile page and you should see articles you posted
* Hit the delete button
* Then confirm the action
* Hit the edit button
* Edit the article and save the changes

#### What are the relevant pivotal tracker stories?
[#162685628](https://www.pivotaltracker.com/story/show/162685628)

#### Screenshots
![image](https://user-images.githubusercontent.com/26769886/53393363-39998a80-39ad-11e9-9a0e-9b7e7f4a9376.png)


![image](https://user-images.githubusercontent.com/26769886/53222519-1efaa500-367f-11e9-9bdd-6544c8821cf5.png)

![image](https://user-images.githubusercontent.com/26769886/53222596-62edaa00-367f-11e9-86af-13c0406c936e.png)

![image](https://user-images.githubusercontent.com/26769886/53222600-67b25e00-367f-11e9-885b-26f390cd2479.png)

![image](https://user-images.githubusercontent.com/26769886/53222609-6f720280-367f-11e9-9aa2-4ee64c6977d1.png)
